crates/sui-types/src/bridge.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::base_types::SequenceNumber;
+use crate::error::SuiResult;
+use crate::object::Owner;
+use crate::storage::ObjectStore;
+use crate::SUI_BRIDGE_OBJECT_ID;
+use move_core_types::ident_str;
+use move_core_types::identifier::IdentStr;
+
+pub const BRIDGE_MODULE_NAME: &IdentStr = ident_str!("bridge");
+pub const BRIDGE_CREATE_FUNCTION_NAME: &IdentStr = ident_str!("create");
+
+pub fn get_bridge_obj_initial_shared_version(
```

Comment:
For clock, this is statically knowable. Is this not the case for this bridge object?

Reply:
The bridge object is created in a End of epoch transaction, so the version is unknown.